pkg/controller/storagecluster/uninstall_reconciler.go

```diff
@@ -46,6 +46,8 @@ func (r *ReconcileStorageCluster) deleteStorageClasses(instance *ocsv1.StorageCl
 			}
 		case errors.IsNotFound(err):
 			reqLogger.Info(fmt.Sprintf("Uninstall: StorageClass %s not found, nothing to do", sc.Name))
+		default:
+			reqLogger.Info(fmt.Sprintf("Uninstall: Unexpected Error %v while getting StorageClass %s", err, sc.Name))
```

Comment:
If I may suggest a change, it is better to keep error at the end of the sentence.

Reply:
```suggestion
			reqLogger.Info(fmt.Sprintf("Uninstall: Error while getting StorageClass %s: %v", err, sc.Name))
```

Reply:
@jarrpa I think the args are reversed.

Reply:
Updated

Reply:
@raghavendra-talur @jarrpa Thanks for the suggestion, I have updated the first commit itself with the changes.